src/Speech/V1/Gapic/SpeechGapicClient.php

```diff
@@ -144,7 +144,7 @@ private static function getGapicVersion()
     /**
      * Return an OperationsClient object with the same endpoint as $this.
      *
-     * @return \Google\GAX\LongRunning\OperationsClient
+     * @return \Google\LongRunning\OperationsClient
```

Comment:
`Google\ApiCore\LongRunning\OperationsClient`

Reply:
Good spot, I will update this upstream